data/map planets.txt

```diff
@@ -1725,7 +1725,7 @@ planet Geyser
 	security 0.3
 	tribute 1300
 		threshold 3500
-		fleet "Large Militia" 20
+		fleet "Small Republic" 20
```

Comment:
I imagine this was set to militia because the spaceport is falling apart from disrepair. Sure, it's right next to Sol, but it's not like it's a great place to garrison a fleet.
> By now many of the hangars have collapsed, and the ones that remain have rugged plastic netting bolted to the ceilings to protect ships from falling fragments of concrete. Only one of the food courts remains active, and although a few of the hotel rooms are still used, the marble fixtures and tile floors that were once so polished and clean are now chipped and cracked and worn down.

Reply:
To me I feel like MZ copied some of the Rim planets when he was originally making the region and forgot to update some of the things, hahaha.

Nonetheless I suppose this is a fair point, but it just seems like such a weird gap that's not really explained well, even if you can make an implication through the spaceport description. I'd think the Republic Navy has a separate garrison from the commercial spaceport.

Reply:
Fair enough.